### PR TITLE
CommandBar updates

### PR DESCRIPTION
**Pull request by teaP**
Visual changes to CommandBar, AppBarButton, and AppBarToggleButton. I removed the CommandBarWithoutRevealStyle since, I figure, the new style IS without reveal.

image

### CODE REVIEW

**Comment by chingucoding** [comment]
Could this cause merge issues with #3852?

**Comment by teaP** [comment]
> Could this cause merge issues with #3852?

Good question. I think probably not... but let's make a couple quick changes on your PR and get it into master, then I can merge it in and check!

**Comment by harvinders** [comment]
It was one area where reveal really shined. I hope there would be a way to enable it if we want in the commandbar.

**Comment by mdtauk** [comment]
> It was one area where reveal really shined. I hope there would be a way to enable it if we want in the commandbar.

Reveal its being depricated with this visual refresh

File: dev/dll/Microsoft.UI.Xaml.vcxproj [hunk 1]
```diff
@@ -698,6 +698,7 @@
     <ItemGroup>
       <FileWrites Include="@(PageRequiringCustomCompilation -> '$(GeneratedFilesDir)Themes\%(Filename).xaml')" />
       <FileWrites Include="@(PageRequiringCustomCompilation -> '$(GeneratedFilesDir)Themes\%(Filename).xbf')" />
+      <FileWrites Include="@(PageRequiringCustomCompilation -> '$(GeneratedFilesDir)Themes\%(Filename).xaml')" />
```
Comment (karenbtlai) [inline dev/dll/Microsoft.UI.Xaml.vcxproj:701]: Is this a duplicate of line 699?

**Comment by ranjeshj** [comment]
> > Could this cause merge issues with #3852?
> 
> Good question. I think probably not... but let's make a couple quick changes on your PR and get it into master, then I can merge it in and check!

That PR is merged now, @teaP you can merge in master to get those changes.

**Comment by teaP** [comment]
Separators look good to me (except the color, but that will be updated).
image

File: dev/dll/Microsoft.UI.Xaml.vcxproj (hunk 1 above)
Comment (teaP) [reply dev/dll/Microsoft.UI.Xaml.vcxproj]: Yes.... weird. I removed it.

**Comment by tashatitova** [comment]
> Separators look good to me (except the color, but that will be updated).
> image

What's that overlay on the light image? Gray background partly covering everything from the bottom?

**Comment by tashatitova** [comment]
Capture

File: dev/CommonStyles/AppBarButton_themeresources.xaml [hunk 2]
```diff
@@ -112,17 +119,15 @@
...
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
         <contract6Present:Setter Property="KeyboardAcceleratorPlacementMode" Value="Hidden" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="AppBarButton">
                     <Grid x:Name="Root"
+                        Background="Transparent"
                         MinWidth="{TemplateBinding MinWidth}"
-                        MaxWidth="{TemplateBinding MaxWidth}"
-                        Background="{TemplateBinding Background}"
```
Comment (tashatitova) [inline dev/CommonStyles/AppBarButton_themeresources.xaml:122]: why do we not need these TemplateBinding's anymore? 

also not sure about the Transparent background (grid is transparent by default)

File: dev/CommonStyles/AppBarToggleButton_themeresources.xaml [hunk 3]
```diff
@@ -218,17 +227,15 @@
...
         <Setter Property="AllowFocusOnInteraction" Value="False" />
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
         <contract6Present:Setter Property="KeyboardAcceleratorPlacementMode" Value="Hidden" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="AppBarToggleButton">
                     <Grid x:Name="Root"
+                        Background="Transparent"
                         MinWidth="{TemplateBinding MinWidth}"
-                        MaxWidth="{TemplateBinding MaxWidth}"
```
Comment (tashatitova) [inline dev/CommonStyles/AppBarToggleButton_themeresources.xaml:227]: same question here

**Comment by teaP** [comment]
> What's that overlay on the light image? Gray background partly covering everything from the bottom?

That is just the background grid of the test app showing through slightly.

**Comment by mdtauk** [comment]
With rounded corners, will the command bar still be flush with the edges of the screen/panel?

I asked this with the new rounded scrollbar because it feels as if it should have some padding, maybe 4px so its close without touching.

File: dev/CommonStyles/AppBarButton_themeresources.xaml (hunk 2 above)
Comment (teaP) [reply dev/CommonStyles/AppBarButton_themeresources.xaml:122]: I moved them to the InnerBorder (Background, Border brush/thickness, CornerRadius).

Technically grid has no background by default, the difference being that items without a background set do not get mouse hover events. :/ That's why I had to set it to transparent.

**Comment by tashatitova** [comment]
> > What's that overlay on the light image? Gray background partly covering everything from the bottom?
> 
> That is just the background grid of the test app showing through slightly.

Oh, so it's that transparent. Didn't know that.

File: dev/CommonStyles/AppBarButton_themeresources.xaml (hunk 2 above)
Comment (tashatitova) [reply dev/CommonStyles/AppBarButton_themeresources.xaml:122]: right, it's true with hover/press situation.

**Review by karenbtlai — APPROVED** [review]

**Pull request merged**